EventListener/ViewResponseListener.php

```diff
@@ -75,16 +76,28 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
             if ($configuration->getStatusCode() && (null === $view->getStatusCode() || Codes::HTTP_OK === $view->getStatusCode())) {
                 $view->setStatusCode($configuration->getStatusCode());
             }
+
+            // BC < 1.8
+            $viewClass = 'FOS\RestBundle\View\View';
+            if (get_class($view) == $viewClass) {
```

Comment:
Yeah this is just to improve the performance as the reflection classes are very slow.